src/plugins/discover/public/application/main/components/top_nav/get_top_nav_links.tsx

```diff
@@ -58,6 +61,43 @@ export const getTopNavLinks = ({
     testId: 'discoverOptionsButton',
   };
 
+  const alerts = {
+    id: 'alerts',
+    label: i18n.translate('discover.localMenu.localMenu.alertsTitle', {
+      defaultMessage: 'Alerts',
+    }),
+    description: i18n.translate('discover.localMenu.alertsDescription', {
+      defaultMessage: 'Alerts',
+    }),
+    run: (anchorElement: HTMLElement) => {
+      if (savedSearch.searchSource.getField('index')?.timeFieldName) {
+        openAlertsPopover({
+          I18nContext: services.core.i18n.Context,
+          anchorElement,
+          searchSource: savedSearch.searchSource,
+          services,
+        });
+      } else {
+        services.toastNotifications.addDanger({
+          title: i18n.translate('discover.alert.errorHeader', {
+            defaultMessage: "Cant't create alert",
```

Comment:
```suggestion
            defaultMessage: "Can't create alert",
```

Reply:
So when the user clicks on Alerts and there's no data view with time fields, the popover is displayed. Functionality wise this is fine since we want to prevent this input. However this way the user also can go to management. So e.g. we could disable the menu point in the case there's no time field? So the user could still navigate to management

Reply:
Oh I just saw in the code, it's displayed to create a new alert rule if there's no timefield.

Reply:
I have disabled menu point in the case there's no time field and added explanation tool tip. See description.